### ISSUE TITLE
Fix `at-rule-no-unknown` false positives for `@position-try`

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#7965]
### What minimal example or steps are needed to reproduce the bug?

```css
@position-try --foo {}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "at-rule-no-unknown": true
  }
}
```

### How did you run Stylelint?

https://stylelint.io/demo

### What did you expect to happen?

no error

### What actually happened?

`Unexpected unknown at-rule "@position-try"`

### Resources

https://developer.mozilla.org/en-US/docs/Web/CSS/@position-try#browser_compatibility

**Comment by Mouvedia** [comment]
I've labeled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.
